airtime_mvc/application/views/scripts/form/preferences_general.phtml

```diff
@@ -12,7 +12,15 @@
 <!--        <button id="logo-remove-btn" value="Remove" class="btn">Remove</button>-->
 
         <div id="Logo-img-container">
-            <img onError="this.onerror = '';this.style.visibility='hidden';$('#logo-remove-btn').hide();$('[id^=stationLogoRemove]').each(function(i,v){v.style.width=v.style.height=v.style.margin=v.style.padding='0px';});" id="logo-img" onload='resizeImg(this, 450, 450);' src="data:image/png;base64,<?php echo $this->element->getView()->logoImg ?>" />
+            <?php
+            $logoImg = $this->element->getView()->logoImg;
+            if ($logoImg === "airtime_logo.png") {
+                $src = "airtime_logo.png";
```

Comment:
Continuing my comment from above... I'd change this logic to be like, if ($logoImg === DEFAULT_LOGO_PLACEHOLDER) then $src = DEFAULT_LOGO_FILE.

where DEFAULT_LOGO_FILE would be a constant defined to "airtime_logo.png" in constants.php.